code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -77,15 +77,13 @@
 			blood_type = "LE"
 		else
 			blood_type = null
-	update_pack_name()
-	update_appearance(UPDATE_ICON)
+	update_appearance(UPDATE_ICON | UPDATE_NAME)
 
-/obj/item/reagent_containers/blood/proc/update_pack_name()
-	if(!labelled)
-		if(blood_type)
-			name = "blood pack - [blood_type]"
-		else
-			name = "blood pack"
+/obj/item/reagent_containers/blood/update_name()
+	. = ..()
+	if(labelled)
+		return
+	name = "blood pack[blood_type ? " - [blood_type]" : ""]"
```

Comment:
This is unrelated to GAGS

Reply:
it is just a code improv that i did so i could rerun tests